### PR TITLE
Daisy e2e: Increase wait period prior to checking logs

### PR DESCRIPTION
**Pull request by EricEdens**
The test `validate_cloud_logging.wf.json` has recently been failing with "BuildFailed: Expected Cloud Logs.". This increases the wait period prior to checking for logs.

The alternative is to use a retry mechanism. I opted against that since one of the cases *doesn't* want logs, and therefore need to wait until the logs would have propagated and become available.

Testing:
* Ran validate_cloud_logging test and verified that it passed.

### CODE REVIEW

File: daisy_integration_tests/scripts/run_daisy_and_check_logs.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ run_daisy() {
 
 verify_logs() {
   echo 'BuildStatus: Waiting for logs to propagate to StackDriver.'
-  sleep 30
+  sleep 120
```
Comment (zoran15) [inline daisy_integration_tests/scripts/run_daisy_and_check_logs.sh:53]: Do we know why is this necessary? What changed yesterday that made this change necessary?

Comment (EricEdens) [reply daisy_integration_tests/scripts/run_daisy_and_check_logs.sh:53]: The timeline of this lines up: https://status.cloud.google.com/incident/google-stackdriver/19004 . That said, I like this longer period since it benefits the case where we don't expect logs to be uploaded.
